### ISSUE TITLE
NeoVintageous in the Command Palette?

### ISSUE THREAD

**Issue by godbout** [NeoVintageous/NeoVintageous#1035]
**Is your feature request related to a problem? Please describe.**

kinda related to a personal problem, yeah? i built an app that adds Vim motions everywhere on macOS. there's of course a way to tell it to back off in apps that have their own Vim systems, like Sublime Text with (the awesome) NeoVintageous. now my app adds Vim motions also in the macOS UI! which means that i can go up and down dropdowns with `j` `k`. now i super miss that in the Sublime Text Command Palette. would be super nice if we could enter Normal Mode in the Palette (even to modify what we're searching for using Vim motions) and being able to go down the list from `j` and `k`.

**Describe the solution you'd like**

when we open the Command Palette, press `esc` (or whatever custom shortcut we set up for NeoVintageous) to enter Normal Mode. then being able to go down the list with `j` and `k`.  

**Describe alternatives you've considered**

crying.

**Additional context**

of course i'm aware that that would only be possible if Sublime Text gives some API to access the Command Palette etc. which is totally possible they don't. still shooting my shot.

**Comment by gerardroche** [comment]
> i can go up and down dropdowns with j k. now i super miss that in the Sublime Text Command Palette. 

`CTRL-j` / `CTRL k` or `CTRL-n` / `CTRL-p` are supported. They are also supported in the other overlays and things like autocomplete.

> would be super nice if we could enter Normal Mode in the Palette (even to modify what we're searching for using Vim motions) and being able to go down the list from j and k.

I don't know that it's possible have modes in the Command Palette, at least not of the top of my head. Maybe it's possible, but it's probably very messy. Typically we use `CTRL-j` / `CTRL k` or `CTRL-n` / `CTRL-p` for things like this.

Take a look at NeoVintageous Files plugin for additional keys support for the sidebar and overlays. Don't forget to install the dependencies Origami and SideBarTools. https://github.com/gerardroche/NeoVintageousFiles

**Comment by godbout** [comment]
> `CTRL-j` / `CTRL k` or `CTRL-n` / `CTRL-p` are supported. They are also supported in the other overlays and things like autocomplete.

yeah i used to use `CTRL-n` and `CTRL-p` before i built kindaVim. but it was still annoying coz it was very inconsistent. working in some apps and not working in others. so i built kV to get `j` `k` working everywhere 😂️ now i don't use `CTRL-n` `CTRL-p` anymore at all, even in the Terminal. history through Vim motions FTW.

didn't know about `CTRL-j` and `CTRL-k` in Sublime tho. seems that would be the closest we can go with for now. thanks.

> I don't know that it's possible have modes in the Command Palette, at least not of the top of my head. Maybe it's possible, but it's probably very messy. Typically we use `CTRL-j` / `CTRL k` or `CTRL-n` / `CTRL-p` for things like this.

i see. a pity. btw just reminding me that NeoVintageous somehow is available in the search fields. when i start Sublime and search for something right away, NeoVintageous starts within those fields lol. anyways just throwing this out in the wild in case it actually makes you wanna check out the Palette hahahahah

> Take a look at NeoVintageous Files plugin for additional keys support for the sidebar and overlays. Don't forget to install the dependencies Origami and SideBarTools. https://github.com/gerardroche/NeoVintageousFiles

ah nice. wasn't aware of that. but tbh my main pet peeve is really the Command Palette. too used to `j`/`k` in Alfred, Spotlight, or whatever menu i have in front of my face. thanks anyways!
